lib/MCToIRLifter.cpp

```diff
@@ -123,13 +123,42 @@ void MCToIRLifter::VisitIndirectJump(const remill::Instruction &inst,
   remill::AddTerminatingTailCall(block, intrinsics.jump);
 }
 
+void MCToIRLifter::VisitConditionalIndirectJump(const remill::Instruction &inst,
```

Comment:
Could you add a comment that describes at a high level what this visitor is doing? I see it makes a conditional branch, but I am curious why and what each side of the branch is logically doing